src/client/helpers/entity.js

```diff
@@ -206,3 +206,7 @@ export function genEntityIconHTMLElement(entityType, size = '', margin = true) {
 			size={size}
 		/>);
 }
+
+export function getSortNameOfDefaultAlias(entity) {
+	return entity.defaultAlias.sortName;
```

Comment:
I'm getting an error when trying to delete an entity (on my local machine):
Cannot read property 'sortName' of undefined

Looks like you're missing an entity.defaultAlias && in there.
Please have a quick look at your recent PRs in case there is another potential issue.

Sorry I didn't catch that beforehand !